azure-devops/run_build.yml

```diff
@@ -47,20 +51,23 @@ jobs:
         buildDirectory: $(Build.ArtifactStagingDirectory)/tools
     - task: BatchScript@1
       displayName: 'Enforce clang-format'
+      timeoutInMinutes: 5
       condition: eq('${{ parameters.targetPlatform }}', 'x64')
       inputs:
         filename: 'azure-devops/enforce-clang-format.cmd'
         failOnStandardError: true
         arguments: '$(Build.ArtifactStagingDirectory)/tools/parallelize/parallelize.exe'

```

Comment:
why even bother with parallelize here? clang format is fast enough as it is.

Reply:
Pre-existing. I've noted your suggestion in https://github.com/microsoft/STL/issues/257#issuecomment-550034871 , thanks.

Reply:
It is not fast, without parallelization it is the most expensive part of the build. Considering the parallelism tech is also going to be used in the test harness if that's broken we need to fix it. I would be dropping everything to investigate right now were I not in Belfast.